integration_test_lite.py

```diff
@@ -151,10 +155,20 @@ async def get_bad_apis(self):
         return [result for result in results if result]
 
 
+def write_log(good_apis, bad_apis):
+    log = {
+        'passed_tests': good_apis,
+        'failed_tests': bad_apis
+    }
+    with open('./log.json', 'w') as f:
```

Comment:
I recommend changing the name from `log.json` to `api-status.json` to avoid the confusion.